modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/TranslationManager.testcase

```diff
@@ -92,6 +92,66 @@ definition {
 		}
 	}
 
+	@description = "LPS-148142. Validates Manage Translations cancel add language by different ways."
+	@priority = "3"
+	test CancelAddLanguageByDifferentWays {
+		task ("Given manage translations modal") {
+			TranslationManagerPortlet.goToManageTranslationsModal(
+				key_currentLocale = "en-us",
+				key_title = "Admin");
+		}
+
+		task ("When add language in Manage Translations table list") {
+			ManageTranslations.addLanguage(key_locale = "zh-CN");
+		}
+
+		task ("Then cancel add a language with cancel button") {
+			Click(locator1 = "Button#CANCEL");
+		}
+
+		task ("Then view language not active in secltor menu") {
+			TranslationManagerPortlet.clickSelectTranslation(
+				key_currentLocale = "en-us",
+				key_title = "Admin");
+
+			AssertElementNotPresent(
+				key_locale = "zh-CN",
+				locator1 = "Translation#DROPDOWN_MENU_ITEM");
+		}
+
+		task ("Then close secltor menu") {
+			TranslationManagerPortlet.clickSelectTranslation(
+				key_currentLocale = "en-us",
+				key_title = "Admin");
+		}
+
+		task ("Given manage translations modal") {
+			TranslationManagerPortlet.goToManageTranslationsModal(
+				key_currentLocale = "en-us",
+				key_title = "Admin");
+		}
+
+		task ("When add language in Manage Translations table list") {
+			ManageTranslations.addLanguage(key_locale = "zh-CN");
+		}
+
+		task ("Then cancel add a language with click somewhere outside the modal") {
+			Click.mouseDownMouseUp(locator1 = "TranslationManager#AUI_TAG");
+
+			AssertElementNotPresent(locator1 = "Modal#MODAL");
+		}
+
+		task ("Then view language not active in secltor menu") {
```

Comment:
is `secltor` a typo?

Reply:
This is an undetected error, I will edit it.